src/test/regress/sql/background_rebalance.sql

```diff
@@ -110,6 +110,27 @@ SELECT public.wait_for_resource_cleanup();
 SELECT 1 FROM citus_remove_node('localhost', :master_port);
 SELECT public.wait_until_metadata_sync(30000);
 
+-- make sure a non-super user can rebalance when there are reference tables to replicate
+CREATE TABLE ref_table(a int primary key);
+SELECT create_reference_table('ref_table');
+
+-- add a new node to trigger replicate_reference_tables task
+SELECT 1 FROM citus_add_node('localhost', :worker_3_port);
+
+SET ROLE non_super_user_rebalance;
+SELECT 1 FROM citus_rebalance_start(shard_transfer_mode := 'force_logical');
```

Comment:
kind of wondering whether we actually want any random user to be able to run a rebalance, but that's a pre-existing issue